assets/js/app.js

```diff
@@ -1,122 +1,147 @@
-// Executes code when DOM has fully loaded
-$(document).ready(function() {
-
-
-// Create the character object to hold the characters data
 const characters = {
 	"John_Snow": {
 		name: "John Snow",
-		healthPoints: 222, 
-		attack: 10, 
+		healthPoints: 222,
+		attack: 10,
 		image: "assets/images/john_snow.jpg",
 		enemyAttackBack: 15
 	},
 
 	"Arya_Stark": {
 		name: "Arya Stark",
-		healthPoints: 333, 
-		attack: 15, 
+		healthPoints: 333,
+		attack: 15,
 		image: "assets/images/arya_stark.jpg",
 		enemyAttackBack: 10
 	},
 
 	"Grey_Worm": {
 		name: "Grey Worm",
-		healthPoints: 445, 
-		attack: 20, 
+		healthPoints: 445,
+		attack: 20,
 		image: "assets/images/Grey_Worm.jpg",
 		enemyAttackBack: 8
 	},
-	
+
 	"Danny_T": {
 		name: "Daenerys Targaryen",
-		healthPoints: 225, 
-		attack: 28, 
+		healthPoints: 225,
+		attack: 28,
 		image: "assets/images/d_t.jpg",
 		enemyAttackBack: 16
 	}
 };
 
-// Create a function that creates the character cards
-function createCharacterCard (character, cardArea) {
-	// Create a div for each character's card containing name, healthPoints, and image.
-	const $characterDiv = $("<div>", {"class": "characters"});
-	$characterDiv.attr("data-name", character.name);
-	const $characterName = $("<p>");
-	$characterName.attr("class", "characterName");
-	$characterName.text(character.name);
-	const $characterImage = $("<img>");
-	$characterImage.attr("class", "characterImage");
-	$characterImage.attr("src", character.image);
-	const $characterHealthPoints = $("<p>");
-	$characterHealthPoints.attr("class", "characterHealthPoints");
-	$characterHealthPoints.text(character.healthPoints);
-	$characterDiv.append($characterName).append($characterHealthPoints).append($characterImage);
-	$(cardArea).append($characterDiv);
-}
-
-function createCharacter () {
-	for(var key in characters) {
-		console.log(characters[key]);
-		createCharacterCard(characters[key], "#characterArea");
+/**
+ * [description]
+ * @return {[type]} [description]
+ */
+const character = (function(){
+	/**
+	 * [createCharacter description]
+	 * @return {[type]} [description]
+	 */
+	function createCharacter() {
+		for(var key in characters) {
```

Comment:
So we've discussed the differences in `for..in` and `for...of`. I forgot to mention that `$.each` is also a viable option here. http://api.jquery.com/jquery.each/

As a general rule, as you begin to develop your own style, you should stick to one thing or the other. You are mixing styles where you use plain JS and jQuery. Consistency is the key to developing your own style.

Reply:
I remember, I need to say if iterable hasOwnProperty of key. Would this be correct? 
`function  createCharacter() {
for(var key in characters) { 
(if characters.hasOwnProperty(key)) console.log(key) 
}
}`